planning/behavior_path_start_planner_module/include/behavior_path_start_planner_module/start_planner_module.hpp

```diff
@@ -191,7 +191,7 @@ class StartPlannerModule : public SceneModuleInterface
 
   std::vector<std::shared_ptr<PullOutPlannerBase>> start_planners_;
   PullOutStatus status_;
-  mutable StartGoalPlannerData start_planner_data_;
+  mutable StartPlannerDebugData start_planner_data_;
```

Comment:
```suggestion
  mutable StartPlannerDebugData debug_data_;
```

Reply:
Thanks!
changed in https://github.com/autowarefoundation/autoware.universe/pull/6101/commits/e6ab77a1bc3464ba3accd53cdc90e0e572394fe4